src/NewRelic.Core/DotnetVersion.cs

```diff
@@ -26,7 +26,8 @@ public enum DotnetCoreVersion
     {
         LessThan30,
         netcoreapp30,
-        GreaterThan30
```

Comment:
Is there any reason why we don't have a `netcoreapp31` version?

Reply:
I don't have the historical context why we didn't have that in the past, but it is good that you pointed this out because, net31 would also fall into the `GreaterThan30` category in the current logic together with `net5`. So to keep thing more consistent from now on, I see 2 ways to address this:

1. Still keeping `GreaterThan30` metrics and assume it will represent `netcoreapp31`. This is simplest.
2. Removing `GreaterThan30` metric and creating `netcoreapp31` metric, this would mean `GreaterThan30` and `netcoreapp31` will now represent `netcoreapp31`.

Reply:
I'm kinda leaning toward option 1 to keep thing simple.

Reply:
Using `GreaterThan30` to represent `netcoreapp31` creates a question unless you know the Agent implementation and that it really means 'greater than 3.0 but not 5.0'. Granted, customers don't (currently) have access to Supportability Metrics, but for other (internal) users (Support, FIT, etc.) `GreaterThan30` doesn't tell the full story. I vote for 2.

Reply:
Discussed with @nr-ahemsath. Decided to go with option 2 as it would reduce the amount the customer has to think about things.